Inference for masking

Jake-Janssen opened the pull request.

Commit:
Config revisions, storing data, adjusting weight and saving locations

Commit:
allowing config.json for weights

Commit:
prep for open source

Commit:
Renamed data directories
Modified import paths
all scripts moved to scripts
inference_config.json now saved off
more data added
slight tuning on confidence threshold
wrote config readme

Commit:
added requirements to the readme

Commit:
Masking inference on point cloud segmentation

mpowelson commented:
We have started using the convention of adding WIP: when the pull request is not ready. This is a carry over from using gitlab. eg your title would be `WIP: Inference for masking`

Commit:
udpated naming convertions
added support for multi-classification training and inference
added training and inference details to the README

Commit:
import path touch ups

Commit:
update to weld config files

Commit:
small change to preprocessing for greyscale images and more useful inference file

Commit:
forgot print statement

Commit:
added config files for other weld detection weights
